server/infra/database/PlanterRepository.ts

```diff
@@ -52,6 +53,7 @@ export default class PlanterRepository extends BaseRepository<Planter> {
       FROM planter
       LEFT JOIN planter_registrations
            ON planter.id = planter_registrations.planter_id
+           AND planter_registrations.created_at = (Select Min(created_at) from planter_registrations planter_registrations.planter_id=planter.id)
```

Comment:
@aayushgauba did you try to run this part of code?

Reply:
@dadiorchen Oh I see what you are saying. Please check my new pull request as that should fix this. Thanks